File: AngularDotNetProject/AngularDotNetProject.Domain/Domain/Headline.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AngularDotNetProject.Domain.Domain
{
    public class Headline
    {
        public int HeadlineId { get; set; }
        public string Name { get; set; }
        public string HighLight { get; set; }
        public string ImageURL { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public List<HeadlineEvent> HeadlineEvents { get; set; }
        public List<SocialNetwork> SocialNetworks { get; set; }
        

    }
}

File: Angular-app/src/app/_models/Headline.ts
import { SocialNetwork } from './SocialNetwork';
import { Event } from './Event';

export interface Headline {
    headlineId: number; 
    name: string; 
    highLight: string; 
    imageURL: string; 
    phone: string; 
    email: string; 
    headlineEvents: Event[]
    socialNetworks: SocialNetwork[]

}

